app/server/lib/DocApi.ts

```diff
@@ -575,7 +584,7 @@ export class DocWorkerApi {
     );
 
     // Responds with an archive of all attachment contents, with suitable Content-Type and Content-Disposition.
-    this._app.get('/api/docs/:docId/attachments/download', canView, withDoc(async (activeDoc, req, res) => {
+    this._app.get('/api/docs/:docId/attachments/archive', canView, withDoc(async (activeDoc, req, res) => {
```

Comment:
A Paul™ approved rename to this endpoint, so we can use it for both POST and GET